corehq/apps/app_manager/templates/app_manager/partials/forms/case_config_advanced.html

```diff
@@ -521,7 +521,7 @@ <h4 class="panel-title panel-title-nolink">
               </h4>
             </div>
             <div class="panel-body">
-                <div id="case-load-accordion" class="panel-group" data-bind="foreach: load_update_cases, sortableList: load_update_cases, sortableListItems: function () { return 'div:not(not-droppable)'; }">
+                <div id="case-load-accordion" class="panel-group" data-bind="foreach: load_update_cases, sortableList: load_update_cases, sortableListSelector: function () { return 'div:not(not-droppable)'; }">
```

Comment:
Confused by this: if this is a selector doesn't it need to be `div:not(.not-droppable)`?

Reply:
It definitely worked without the . but agreed the explicit class is better https://github.com/dimagi/commcare-hq/pull/23689/commits/297023fc419c5ea5aec4f1446a8cae650cf92dcb